.gitlab/prepare-oci-package.sh

```diff
@@ -16,6 +16,11 @@ if [ -z "$ARCH" ]; then
   ARCH=amd64
 fi
 
+if [ "$OS" != "linux" ]; then
+  echo "Only linux packages are supported. Exiting"
+  exit 0
+fi
```

Comment:
Interesting, so it's the presence of some artifacts that trigger the build of the Windows OCI images, right?

Reply:
Yes. Checks for the presence of files in `sources`